features/plate_destruction_verification.feature

```diff
@@ -8,72 +8,26 @@ Feature: Verify destruction of plates
       And all this is happening at "2015-05-05T13:45:00+01:00"
 
   Scenario Outline: Destroying various plates
-    Given the search with UUID "00000000-0000-0000-0000-000000000001" for barcodes "<plate_1>, <plate_2>" returns the following JSON:
-      """
-      {
-          "searches": [{
-              "uuid": "00000000-1111-2222-3333-444444444444",
-              "barcode": {
-                "ean13": "old_plate",
-                "machine": "old_plate"
-              },
-              "plate_purpose": { "lifespan":30, "uuid":"00000000-2222-2222-3333-444444444444" },
-              "created_at":"2015-01-01T13:45:00+01:00"
-          },
-          {
-              "uuid": "00000000-1111-2222-3333-444444444445",
-              "barcode": {
-                "ean13": "new_plate",
-                "machine": "new_plate"
-              },
-              "plate_purpose": {"lifespan":30, "uuid":"00000000-2222-2222-3333-444444444444" },
-              "created_at":"2015-05-01T13:45:00+01:00"
-          },
-          {
-              "uuid": "00000000-1111-2222-3333-444444444446",
-              "barcode": {
-                "ean13": "old_indestructable_plate",
-                "machine": "old_indestructable_plate"
-              },
-              "plate_purpose": { "lifespan":null, "name":"immortal", "uuid":"00000000-2222-2222-3333-444444444446"  },
-              "created_at":"2015-01-01T13:45:00+01:00"
-          },
-          {
-              "uuid": "00000000-1111-2222-3333-444444444447",
-              "barcode": {
-                "ean13": "other_old_plate",
-                "machine": "other_old_plate"
-              },
-              "plate_purpose": {"lifespan":30, "uuid":"00000000-2222-2222-3333-444444444444" },
-              "created_at":"2015-01-01T13:45:00+01:00"
-          }],
-          "uuids_to_ids": {
-              "00000000-1111-2222-3333-444444444444": 194216,
-              "00000000-1111-2222-3333-444444444445": 194217,
-              "00000000-1111-2222-3333-444444444446": 194218,
-              "00000000-1111-2222-3333-444444444447": 194219
-          }
-      }
-      """
+    Given I can retrieve the labware with barcodes "<plate_1>, <plate_2>" and lifespans "<lifespan_1>, <lifespan_2>" and ages "<age_1>, <age_2>" and existence "<exists_1>, <exists_2>"
```

Comment:
worth testing with a tube? or mix of plates and tubes?

Reply:
I did write one, but then removed it as the test was identical - because I've changed it to get 'labware' through the API, rather than 'plates', the response is identical either way. 

So the real test is whether the API calls work (which they do) - something I could only test with an integration test